XiEditor/EditViewController.swift

```diff
@@ -805,6 +814,22 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         }
         self.unifiedTitlebar = { self.unifiedTitlebar }()
     }
+    
+    public func availableLanguagesChanged(_ languages: [String]) {
+        let languagesMenu = NSApplication.shared.mainMenu!.item(withTitle: "Debug")!.submenu!.item(withTitle: "Language")!.submenu!;
```

Comment:
I always get a bit scared whenever I see this many `!`s. Can it happen that this RPC arrives before the menu has loaded, but the view controller hasn't? I'd _hope_ not.

In any case, I think it's probably fine, at least this way we'll find out if/when it happens.. 😏

Reply:
> Can it happen that this RPC arrives before the menu has loaded, but the view controller hasn't?
I'm not sure about that, but it never happened on my machine 😅 
Also, the same goes for `available_themes`, and it's been in the code for a while, so I think we're good here